conf/config.yaml.sample

```diff
@@ -23,26 +35,33 @@ cortx:
   utils:
     message_bus_backend: kafka
   s3:
+    iam:
```

Comment:
There is a 3rd endpoint added for s3 w.r.t. ConfigMap
  s3:
    iam:
      endpoints:
      - https://iam.seagate.com:9443
    data:
      endpoints:
      - http://s3.seagate.com:80
      - https://s3.seagate.com:443
    internal:
      endpoints:
      - http://s3.seagate.com:28049

Reply:
internal:
endpoints:
- http://s3.seagate.com:28049

Added above one in all cluster.yaml files.